setup.py

```diff
@@ -35,6 +35,18 @@ def run(self):
         for ext in self.extensions:
             self.build_extension(ext)
 
+    def get_ext_filename(self, ext_name):
```

Comment:
Overwritten temporarily since it looks like the `build_wheel` step does not have the same env vars in pyoide than the earlier steps during build.

Wheel packaging still does not pick up the `openpmd_api_cxx` shared lib though...